chore(CHANGELOG): updato to v11.0.0 (#1779)
bpmn-io/bpmn-js: CHANGELOG.md

___Note:__ Yet to be released changes appear here._

## 11.0.0
* `FEAT`: integrate new popup menu UI ([#1776](https://github.com/bpmn-io/bpmn-js/pull/1776))
* `DEPS`: update to diagram-js@11.1.0 ([#1776](https://github.com/bpmn-io/bpmn-js/pull/1776))

> ### Breaking Changes
> * New popup menu UI introduced with `diagram-js` update. See `diagram-js` breaking changes and how to migrate to new version [here](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#breaking-changes).

Reviewer: ```suggestion
### Breaking Changes
* New popup menu UI introduced with `diagram-js` update. [See `diagram-js` breaking changes and migration guide](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#breaking-changes).
```

> * New popup menu UI introduced with `diagram-js` update. See `diagram-js` breaking changes and how to migrate to new version [here](https://github.com/bpmn-io/diagram-js/blob/develop/CHANGELOG.md#breaking-changes).

Reviewer: I think this makes. Let's just use meaningful content of the link (not "here").
